src/wordpress-importer.php

```diff
@@ -1224,6 +1221,10 @@ function cmpr_strlen( $a, $b ) {
 } // class_exists( 'WP_Importer' )
 
 function wordpress_importer_init() {
+	if ( ! defined( 'WP_LOAD_IMPORTERS' ) ) {
```

Comment:
Should we take the opportunity to make sure the value of the constant is truthy here as well?
```suggestion
	if ( ! ( defined( 'WP_LOAD_IMPORTERS' ) && WP_LOAD_IMPORTERS ) ) {
```

Reply:
Yes, that seems like the responsible thing to do, just in case anyone ever had a reason to explicitly not load importer code.